src/types.ts

```diff
@@ -154,6 +154,14 @@ export type IAzureDataSourceStatefulProviderProps = {
   options?: DataSourceOptions
   events?: IAzureMapDataSourceEvent | any
   dataFromUrl?: string
+  collection?:
+    | atlas.data.FeatureCollection
+    | atlas.data.Feature<atlas.data.Geometry, any>
+    | atlas.data.Geometry
+    | atlas.data.GeometryCollection
+    | Shape
+    | Array<atlas.data.Feature<atlas.data.Geometry, any> | atlas.data.Geometry | Shape>
```

Comment:
we should have type for atlas.data.Feature<atlas.data.Geometry, any>

Reply:
azure-map-controls use this type